fundamentals/fundamentals-bio/.jvm/src/test/scala/izumi/functional/bio/test/BIOMonixLawsTest.scala

```diff
@@ -0,0 +1,18 @@
+package izumi.functional.bio.test
+
+import cats.effect.laws.ConcurrentLaws
+import cats.effect.laws.discipline.ConcurrentTests
+import cats.effect.{Concurrent, ContextShift}
+import izumi.functional.bio.catz._
+import izumi.functional.bio.env.MonixEnv
+
+class BIOMonixLawsTest extends CatsLawsTestBase with MonixEnv {
+    val concurrentTestsMonix = new ConcurrentTests[monix.bio.Task] {
+      override val laws = new ConcurrentLaws[monix.bio.Task] {
+        val F = Concurrent[monix.bio.Task]
```

Comment:
It is not obvious that it uses a converted instance for `Concurrent` instead of Monix's own instance available by default from implicit scope. If `import izumi.functional.bio.catz._` line is removed, this code will continue compiling, but would use the wrong instance... So I think better to refer to `BIOAsyncForkToConcurrent` directly here
```suggestion
  val concurrentTestsMonix = new ConcurrentTests[monix.bio.Task] {
    override val laws = new ConcurrentLaws[monix.bio.Task] {
      val F = Concurrent[monix.bio.Task](BIOAsyncForkToConcurrent)
      val contextShift = ContextShift[monix.bio.Task](monix.bio.IO.contextShift)
    }
  }
```